Arriba/Arriba.Test/Model/TableTestsLarge.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using Arriba.Extensions;
+using Arriba.Model;
+using Arriba.Model.Aggregations;
+using Arriba.Model.Column;
+using Arriba.Model.Expressions;
+using Arriba.Model.Query;
+using Arriba.Serialization;
+using Arriba.Structures;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Arriba.Test.Model
+{
+    [TestClass]
+    public class TableTestsLarge
+    {
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            ColumnFactory.ResetColumnCreators();
+        }
+
+        [TestMethod]
+        public void LargeTable_SelectBasic()
+        {
+            ITable table = new Table("Sample", 1000000);
```

Comment:
yeah.  I did this earlier to make sure I had more partitions than CPU cores but that was before I thought of the max parallelism idea.